web-marketplace/src/pages/CreditClassDetails/CreditClassDetails.AdditionalInfo.tsx

```diff
@@ -54,36 +57,45 @@ const AdditionalInfo = <T extends CreditClassMetadataLD>({
     >
       {metadata && (
         <Body size="lg" sx={{ pb: 7 }}>
-          {CREDIT_CLASS_DETAILS_ADDITIONAL_INFO_HELPER_TEXT}
+          {_(CREDIT_CLASS_DETAILS_ADDITIONAL_INFO_HELPER_TEXT)}
         </Body>
       )}
       <Grid container spacing={8}>
         <Grid item xs={12} sm={6} sx={{ flexDirection: 'column' }}>
           <Box sx={{ display: 'flex', alignItems: 'center' }}>
             <Label size="xs" sx={{ mr: 1 }}>
-              class id
+              <Trans>class id</Trans>
             </Label>
             <InfoTooltipWithIcon
-              title={CREDIT_CLASS_DETAILS_ADDITIONAL_INFO_CLASS_ID_TOOLTIP}
+              title={_(CREDIT_CLASS_DETAILS_ADDITIONAL_INFO_CLASS_ID_TOOLTIP)}
               outlined
             />
           </Box>
           <Body size="xl">{classId}</Body>
         </Grid>
         <MetaDetail label="registry" value={sourceRegistry} />
         <MetaDetail
-          label="carbon offset standard"
+          label={_(msg`carbon offset standard`)}
           value={carbonOffsetStandard}
         />
         <ApprovedMethodologiesList
           methodologyList={metadata?.['regen:approvedMethodologies']}
         />
-        <MetaDetail label="project activities" value={projectActivities} />
-        <MetaDetail label="sectoral scopes" value={sectoralScopes} />
-        <MetaDetail label="Tokenization Source" value={tokenizationSource} />
-        <MetaDetail label="ecosystem type" value={ecosystemTypes} />
-        <MetaDetail label="verification method" value={verificationMethod} />
-        <MetaDetail label="measured GHGs" value={measuredGHGs} />
+        <MetaDetail
+          label={_(msg`project activities`)}
+          value={projectActivities}
+        />
+        <MetaDetail label={_(msg`sectoral scopes`)} value={sectoralScopes} />
+        <MetaDetail
+          label={_(msg`Tokenization Source`)}
+          value={tokenizationSource}
+        />
+        <MetaDetail label={_(msg`ecosystem type`)} value={ecosystemTypes} />
+        <MetaDetail
+          label={_(msg`verification method`)}
+          value={verificationMethod}
+        />
+        <MetaDetail label={_(msg`measured GHGs`)} value={measuredGHGs} />
```

Comment:
I guess it's good to note that for `unknowFields` labels (from line 102), there will remain untranslated or do you think there could be a way to achieve that with AI?

Reply:
Yes, we can't translate that directly on the front end. In our next meeting, we should start discussing the different strategies (one-time script, on-demand, ...) for translating this type of data.